

syncio.error.client = {
    OBJECT_NOT_FOUND: -1,
    ALREADY_SYNCED: -2,
};


syncio.error.server = {
    '-1': 'Object "{{2}}" not found',
    '-2': 'You are already synced to the object "{{2}}"',
};